client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/protocol/Codec20.java

```diff
@@ -87,6 +87,13 @@ public void writeExpirationParams(ByteBuf buf, long lifespan, TimeUnit lifespanT
       ByteBufUtil.writeVInt(buf, maxIdleSeconds);
    }
 
+   @Override
+   public void writeBloomFilter(ByteBuf buf, int bloomFilterBits) {
```

Comment:
I would rather disable the bloom filter in `org.infinispan.client.hotrod.RemoteCacheManager#createRemoteCache(java.lang.String)` since you have access to the version than add a new method to the codec.

Reply:
I personally prefer to keep versioned stuff in the Codec. That is what we have been doing recently at least. It ends up being a single place for version specific stuff, to me at least.